packages/loader/container-loader/src/loader.ts

```diff
@@ -366,7 +366,10 @@ export class Loader implements IHostLoader {
     public async request(request: IRequest): Promise<IResponse> {
         return PerformanceEvent.timedExecAsync(this.logger, { eventName: "Request" }, async () => {
             const resolved = await this.resolveCore(request);
-            return resolved.container.request({ url: `${resolved.parsed.path}${resolved.parsed.query}` });
+            return resolved.container.request({
+                ...request,
```

Comment:
Opinion: This is probably ok, but I would lean towards more explicitly copying just the headers. As-is, if/when request gets additional stuff on it (whether via `IRequest` type or just patched on), it will be copied over, which may not be correct but won't be obvious.

Reply:
The idea is that only the url is updated by the loader since it has to remove the container specific part. The rest should be sent as-is to the container. The call to container.request should be no different than calling container.request directly with the request (minus the container id in the url part).

Reply:
Ok that makes sense. Thanks.